curations/nuget/nuget/-/StackExchange.Redis.yaml

```diff
@@ -42,24 +42,77 @@ revisions:
   1.2.6:
     licensed:
       declared: MIT
+  2.0.0-alpha.465:
+    licensed:
+      declared: MIT
   2.0.495:
     licensed:
       declared: MIT
+  2.0.510:
+    licensed:
+      declared: MIT
   2.0.513:
     licensed:
       declared: MIT
   2.0.519:
     licensed:
       declared: MIT
+  2.0.544:
+    licensed:
+      declared: MIT
+  2.0.558:
+    described:
+      releaseDate: '2019-04-03'
+      sourceLocation:
+        name: stackexchange.redis
+        namespace: stackexchange
+        provider: github
+        revision: v2.0.558
+        type: git
+        url: 'https://github.com/stackexchange/stackexchange.redis/commit/v2.0.558'
```

Comment:
This is not a valid link. The repo does not have a tagged version for 2.0.558.

Reply:
@capfei - I agree with deleting the url from the PR.